browser/webRTC/client.js

```diff
@@ -1,7 +1,22 @@
 /** You should probably use a different stun server doing commercial stuff **/
 /** Also see: https://gist.github.com/zziuni/3741933 **/
 const ICE_SERVERS = [
-  { url: 'stun:stun.l.google.com:19302' }
+  { url: 'stun:stun.l.google.com:19302' },
+  {
+    url: 'turn:numb.viagenie.ca',
+    credential: 'muazkh',
+    username: 'webrtc@live.com'
+  },
+  {
+    url: 'turn:192.158.29.39:3478?transport=udp',
```

Comment:
pinged all of these just to sanity check that they're live, and they responded as expected.